decrypt-aspera/tests/checker.nf

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env nextflow
+
+/*
+  Copyright (c) 2022, Your Organization Name
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to deal
+  in the Software without restriction, including without limitation the rights
+  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in all
+  copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+  SOFTWARE.
+
+  Authors:
+    Edmund Su
+*/
+
+/*
+ This is an auto-generated checker workflow to test the generated main template workflow, it's
+ meant to illustrate how testing works. Please update to suit your own needs.
+*/
+
+/********************************************************************/
+/* this block is auto-generated based on info from pkg.json where   */
+/* changes can be made if needed, do NOT modify this block manually */
+nextflow.enable.dsl = 2
+version = '0.1.0'  // package version
+
+container = [
+    'ghrc.io': 'ghrc.io/icgc-argo/argo-data-submission.decrypt-aspera'
+]
+default_container_registry = 'ghrc.io'
+/********************************************************************/
+
+// universal params
+params.container_registry = ""
+params.container_version = ""
+params.container = ""
+
+params.cpus = 1
+params.mem = 1  // GB
+params.publish_dir = ""  // set to empty string will disable publishDir
+
+// tool specific parmas go here, add / change as needed
+params.input_file = ""
+params.expected_output = "./expected/mystery_contents.txt"
+
+process decryptAspera {
```

Comment:
why not import the process
```
include { decryptAspera } from '../main'
```

Reply:
Added import
https://github.com/icgc-argo/argo-data-submission/blob/db4eda6d6a8adb91ed22537142c7f4f4aa70310b/decrypt-aspera/tests/checker.nf#L58